src/pyscaffold/shell.py

```diff
@@ -8,12 +8,13 @@
 import subprocess
 import sys
 from pathlib import Path
-from typing import Callable, Optional, Union
+from typing import Callable, Iterator, Optional, Union
 
 from .exceptions import ShellCommandException
 from .log import logger
 
 PathLike = Union[str, os.PathLike]
+EDITORS = ("sensible-editor", "nvim", "vim", "nano", "subl", "code", "notepad", "vi")
```

Comment:
Should we have a comment here for the Sphinx module reference?

Reply:
I will add that, thank you!